Error when running `nx g @nrwl/nest:service` [Nx 11.0.2]

<!-- Please do your best to fill out all of the sections below! -->

## Current Behavior
<!-- What is the behavior that currently you experience? -->
When I run `nx g @nrwl/nest:service` I get an error:
```
Unable to resolve @nrwl/nest:service.
Cannot find module nest-demo-workspace/node_modules/@nrwl/nest/src/schematics/nestjs-schematics/nestjs-schematics#service'
```

## Expected Behavior
<!-- What is the behavior that you expect to happen? -->
<!-- Is this a regression? .i.e Did this used to be the behavior at one point?  -->
I want to generate NestJS service according to [this documentation](https://nx.dev/latest/node/plugins/nest/generators/service).

## Steps to Reproduce
<!-- Help us help you by making it easy for us to reproduce your issue! -->

<!-- Can you reproduce this on https://github.com/nrwl/nx-examples? -->
<!-- If so, open a PR with your changes and link it below. -->
<!-- If not, please provide a minimal Github repo -->
<!-- At the very least, provide as much detail as possible to help us reproduce the issue -->

### Failure Logs
<!-- Please include any relevant log snippets or files here. -->
```
➜  nest-demo-workspace git:(master) ✗ nx g @nrwl/nest:service
Unable to resolve @nrwl/nest:service.
Cannot find module 'nest-demo-workspace/node_modules/@nrwl/nest/src/schematics/nestjs-schematics/nestjs-schematics#service'
Require stack:
- nest-demo-workspace/node_modules/@nrwl/tao/src/shared/workspace.js
- nest-demo-workspace/node_modules/@nrwl/tao/src/commands/generate.js
- nest-demo-workspace/node_modules/@nrwl/tao/index.js
-nest-demo-workspace/node_modules/@nrwl/cli/lib/init-local.js
- nest-demo-workspace/node_modules/@nrwl/cli/bin/nx.js
- .nvm/versions/node/v14.15.0/lib/node_modules/@nrwl/cli/bin/nx.js
```

### Environment
<!-- It's important for us to know the context in which you experience this behavior! -->
<!-- Please paste the result of `nx report` below! -->
```
  Node : 14.15.0
  OS   : darwin x64
  npm  : 6.14.8

  nx : 11.0.2
  @nrwl/angular : Not Found
  @nrwl/cli : 11.0.2
  @nrwl/cypress : Not Found
  @nrwl/devkit : 11.0.2
  @nrwl/eslint-plugin-nx : 11.0.2
  @nrwl/express : Not Found
  @nrwl/jest : 11.0.2
  @nrwl/linter : 11.0.2
  @nrwl/nest : 11.0.2
  @nrwl/next : Not Found
  @nrwl/node : 11.0.2
  @nrwl/react : Not Found
  @nrwl/schematics : Not Found
  @nrwl/tao : 11.0.2
  @nrwl/web : Not Found
  @nrwl/workspace : 11.0.2
  typescript : 4.0.5
```

--- a/packages/tao/src/shared/workspace.ts
+++ b/packages/tao/src/shared/workspace.ts
@@ -196,11 +196,9 @@ export class Workspaces {
       const executorsDir = path.dirname(executorsFilePath);
       const schemaPath = path.join(executorsDir, executorConfig.schema || '');
       const schema = JSON.parse(fs.readFileSync(schemaPath).toString());
-      const module = require(path.join(
-        executorsDir,
-        executorConfig.implementation
-      ));
-      const implementation = module.default;
+      const [modulePath, exportName] = executorConfig.implementation.split('#');
+      const module = require(path.join(executorsDir, modulePath));
+      const implementation = module[exportName || 'default'];
       return { schema, implementation };
     } catch (e) {
       throw new Error(
@@ -221,13 +219,13 @@ export class Workspaces {
         generatorsJson.schematics)[normalizedGeneratorName];
       const schemaPath = path.join(generatorsDir, generatorConfig.schema || '');
       const schema = JSON.parse(fs.readFileSync(schemaPath).toString());
-      const module = require(path.join(
-        generatorsDir,
-        generatorConfig.implementation
-          ? generatorConfig.implementation
-          : generatorConfig.factory
-      ));
-      const implementation = module.default;
+      generatorConfig.implementation =
+        generatorConfig.implementation || generatorConfig.factory;
+      const [modulePath, exportName] = generatorConfig.implementation.split(
+        '#'
+      );
+      const module = require(path.join(generatorsDir, modulePath));
+      const implementation = module[exportName || 'default'];
       return { schema, implementation };
     } catch (e) {
       throw new Error(

